apple/RNGestureHandlerDirection.h

```diff
@@ -1,8 +1,33 @@
 #import <Foundation/Foundation.h>
 
+#ifndef RNGestureHandlerDirection_h
+#define RNGestureHandlerDirection_h
+
 typedef NS_ENUM(NSInteger, RNGestureHandlerDirection) {
   RNGestureHandlerDirectionRight = 1,
   RNGestureHandlerDirectionLeft = 2,
   RNGestureHandlerDirectionUp = 4,
   RNGestureHandlerDirectionDown = 8,
+  RNGestureHandlerDirectionUpLeft = 16,
+  RNGestureHandlerDirectionUpRight = 32,
+  RNGestureHandlerDirectionDownLeft = 64,
+  RNGestureHandlerDirectionDownRight = 128,
+};
+
+static NSInteger diagonalDirections[] = {
+    RNGestureHandlerDirectionRight,
+    RNGestureHandlerDirectionLeft,
+    RNGestureHandlerDirectionUp,
+    RNGestureHandlerDirectionDown,
 };
+
+static NSInteger axialDirections[] = {
+    RNGestureHandlerDirectionUpLeft,
+    RNGestureHandlerDirectionUpRight,
+    RNGestureHandlerDirectionDownLeft,
+    RNGestureHandlerDirectionDownRight,
+};
```

Comment:
Shouldn't these be named the other way around?

```suggestion
static NSInteger axialDirections[] = {
    RNGestureHandlerDirectionRight,
    RNGestureHandlerDirectionLeft,
    RNGestureHandlerDirectionUp,
    RNGestureHandlerDirectionDown,
};

static NSInteger diagonalDirections[] = {
    RNGestureHandlerDirectionUpLeft,
    RNGestureHandlerDirectionUpRight,
    RNGestureHandlerDirectionDownLeft,
    RNGestureHandlerDirectionDownRight,
};
```

Reply:
You're absolutely right! Changed in [1570c9e](https://github.com/software-mansion/react-native-gesture-handler/pull/3028/commits/1570c9e5031cf4ad81ca6c28499986c6d98643dd)